_std/defines/time.dm

```diff
@@ -10,32 +10,54 @@
 // during sunrise/sunset
 
 // #define OCEAN_LIGHT  rgb(0.160 * 255, 0.60 * 255, 1.00 * 255, 0.65 * 255)
-#if defined(MAP_OVERRIDE_NADIR)
-	#define OCEAN_LIGHT  rgb(0,0,50)
-#elif (BUILD_TIME_HOUR == 0) || (BUILD_TIME_HOUR - 12 == 0)
-	#define OCEAN_LIGHT  rgb(0.160 * 255, 0.60 * 255, 1.00 * 255, 0.65 * 255)
-#elif (BUILD_TIME_HOUR == 1) || (BUILD_TIME_HOUR - 12 == 1)
-	#define OCEAN_LIGHT  rgb(0.160 * 236, 0.60 * 236, 1.00 * 236, 0.65 * 255)
-#elif (BUILD_TIME_HOUR == 2) || (BUILD_TIME_HOUR - 12 == 2)
-	#define OCEAN_LIGHT  rgb(0.160 * 187, 0.60 * 187, 1.00 * 187, 0.65 * 255)
-#elif (BUILD_TIME_HOUR == 3) || (BUILD_TIME_HOUR - 12 == 3)
-	#define OCEAN_LIGHT  rgb(0.160 * 125, 0.60 * 125, 1.00 * 125, 0.65 * 255)
-#elif (BUILD_TIME_HOUR == 4) || (BUILD_TIME_HOUR - 12 == 4)
-	#define OCEAN_LIGHT  rgb(0.160 *  63, 0.60 *  63, 1.00 *  63, 0.65 * 255)
-#elif (BUILD_TIME_HOUR == 5) || (BUILD_TIME_HOUR - 12 == 5)
-	#define OCEAN_LIGHT  rgb(0.160 *  18, 0.60 *  18, 1.00 *  18, 0.65 * 255)
-#elif (BUILD_TIME_HOUR == 6) || (BUILD_TIME_HOUR - 12 == 6)
-	#define OCEAN_LIGHT  rgb(0.160 *   0, 0.60 *   0, 1.00 *   0, 0.65 * 255)
-#elif (BUILD_TIME_HOUR == 7) || (BUILD_TIME_HOUR - 12 == 7)
-	#define OCEAN_LIGHT  rgb(0.160 *  18, 0.60 *  18, 1.00 *  18, 0.65 * 255)
-#elif (BUILD_TIME_HOUR == 8) || (BUILD_TIME_HOUR - 12 == 8)
-	#define OCEAN_LIGHT  rgb(0.160 *  63, 0.60 *  63, 1.00 *  63, 0.65 * 255)
-#elif (BUILD_TIME_HOUR == 9) || (BUILD_TIME_HOUR - 12 == 9)
-	#define OCEAN_LIGHT  rgb(0.160 * 125, 0.60 * 125, 1.00 * 125, 0.65 * 255)
-#elif (BUILD_TIME_HOUR == 10) || (BUILD_TIME_HOUR - 12 == 10)
-	#define OCEAN_LIGHT  rgb(0.160 * 187, 0.60 * 187, 1.00 * 187, 0.65 * 255)
-#elif (BUILD_TIME_HOUR == 11) || (BUILD_TIME_HOUR - 12 == 11)
-	#define OCEAN_LIGHT  rgb(0.160 * 236, 0.60 * 236, 1.00 * 236, 0.65 * 255)
+#ifdef MAP_OVERRIDE_NADIR
+	// nadir has an 8 hour rotation, but is somewhat evenly lit by both shidd and fugg, as it's quite far from typhone (royal rings district)
+	// the redness increases as shidd rises, and the blueness (and greenness) increases as fugg rises. Still pretty dark most of the time though.
+	// shidd and fugg are on opposite sides of the sky, so as one sets, the other rises.
+	// #define OCEAN_LIGHT  rgb(0,0,50)
+	#if (BUILD_TIME_HOUR == 0) || (BUILD_TIME_HOUR - 8 == 0) || (BUILD_TIME_HOUR - 16 == 0)
+		#define OCEAN_LIGHT  rgb(0.10 * 128, 0.10 * 170, 1.00 *  75, 0.65 * 255)
+	#elif (BUILD_TIME_HOUR == 1) || (BUILD_TIME_HOUR - 12 == 1) || (BUILD_TIME_HOUR - 16 == 1)
+		#define OCEAN_LIGHT  rgb(0.10 *  64, 0.10 * 191, 1.00 *  88, 0.65 * 255)
+	#elif (BUILD_TIME_HOUR == 2) || (BUILD_TIME_HOUR - 12 == 2) || (BUILD_TIME_HOUR - 16 == 2)
+		#define OCEAN_LIGHT  rgb(0.10 *   0, 0.10 * 255, 1.00 * 100, 0.65 * 255) // noon (fugg) rgb(0,26,100), quite bluey
+	#elif (BUILD_TIME_HOUR == 3) || (BUILD_TIME_HOUR - 12 == 3) || (BUILD_TIME_HOUR - 16 == 3)
+		#define OCEAN_LIGHT  rgb(0.10 *  64, 0.10 * 191, 1.00 *  88, 0.65 * 000)
```

Comment:
Why is there 0.65 * 000 on this one line?

Reply:
oh that isn't supposed to be like that, whoops